grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -32,6 +32,8 @@ import org.pih.warehouse.requisition.RequisitionType
 import org.pih.warehouse.shipping.Shipment
 import org.pih.warehouse.shipping.ShipmentStatusCode
 
+import javax.xml.bind.ValidationException
```

Comment:
I don't see this class being used anywhere in the changes, is it necessary?

Reply:
Uh, I forgot to remove it 🫢